### PR TITLE
Drop the catgets library and facility from packages being offered.  m…

### PR DESCRIPTION
**Pull request by JPeterMugaas**
…sys2-runtime 2.10 now has it's own facility and indicate that msys2-runtime replaces catgets

Todo: figure out what to do about the "man" package as that package depends upon the catgets library and will to link with catgets.  I would suggest replacing "man" with "man-db" for consistanstancy with Cygwin and Archlinux.

### CODE REVIEW

File: msys2-runtime/PKGBUILD [hunk 1]
```diff
@@ -21,6 +21,7 @@ makedepends=('cocom'
              'gettext-devel'
              'libiconv-devel'
              'diffutils')
+replaces=('catgets')
```
Comment (Alexpux) [inline msys2-runtime/PKGBUILD]: Remove this line as this must be in package procedure.

File: msys2-runtime/PKGBUILD [hunk 2]
```diff
@@ -4,7 +4,7 @@
 pkgbase=msys2-runtime
 pkgname=('msys2-runtime' 'msys2-runtime-devel')
 pkgver=2.10.0
-pkgrel=1
+pkgrel=3
```
Comment (Alexpux) [inline msys2-runtime/PKGBUILD]: Why 3 not 2

**Comment by Alexpux** [comment]
man-db is already in repo

File: msys2-runtime/PKGBUILD (hunk 2 above)
Comment (JPeterMugaas) [reply msys2-runtime/PKGBUILD]: I went ahead and fixed it.

**Comment by JPeterMugaas** [comment]
Okay.  Then the "man" package has to be rebuilt to remove the catgets dependency.  It was linking to it statically. I haven't gotten to that because I'm working on Ruby 2.5.  But ruby 2.4 needs to be rebuilt at the least because that too depended upon the libssp in gcc-libs.

**Comment by Alexpux** [comment]
You dont fix one more comment

**Pull request merged**